graph_algorithms/algorithms/generated/shortest_ss_pos_wt_file.gsql

```diff
@@ -0,0 +1,75 @@
+# Algorithm for Single-Source Shortest Path with positive weights on edges 
+# The path parameter let user choose whether calculate path or not
+
+CREATE QUERY shortest_ss_pos_wt_file (VERTEX v, BOOL display, BOOL path, STRING filepath) FOR GRAPH social {
```

Comment:
remove display, path. Print the whole path by default.

Reply:
Shall l totally remove display for every query? There is also display for queries like Closeness Centrality. Or just keep the display choice for ACCUM version?